src/Components/GameOver.tsx

```diff
@@ -37,11 +38,15 @@ const GameOver = () => {
   const {gameDispatch} = useGame();
   const gameOver = useGameOverDocState();
   const gameWon = useGameWonDocState();
+  const { setMusicOn, soundOn, playCheeringSound, playLoseSound } = useAudio();
 
   useEffect(() => {
-    if (gameOver && gameWon) playCheering();
-    else if (gameOver && !gameWon) playLose();
-  }, [gameOver, gameWon])
+    if (!soundOn) return;
+    if (!gameOver) return;
+
+    if (gameWon) playCheeringSound() && setMusicOn(false);
+    else playLoseSound() && setMusicOn(false);
+  }, [gameOver, gameWon, soundOn])
```

Comment:
mm we defs dont want this soundOn in dependency.

Reply:
ahhh yep i have removed it